terraform/modules/forest_node/variable.tf

```diff
@@ -0,0 +1,77 @@
+variable "image" {
+  description = "The ID of the AMI to use for the Droplet"
+  type        = string
+}
+
+variable "name" {
+  description = "The name of Forest Droplet"
+  type        = string
+}
+
+variable "observability_name" {
+  description = "The name of the observability Droplet"
+  type        = string
+}
+
+variable "region" {
+  description = "The region where resources will be created"
+  type        = string
+}
+
+variable "size" {
+  description = "The size of the EC2 instance to launch"
+  type        = string
+}
+
+variable "protocol" {
+  description = "The protocol to use for the connection."
+  type        = string
+}
+
```

Comment:
When can this be anything other than `tcp`?

Reply:
Well, there is UDP and ICMP which one is used for DNS requests settings and stuff and the other is if there is a need for ping requests or something.